### DISCUSSION TITLE
Library building and testing

### DISCUSSION THREAD

**Discussion by ric4v1** [dolanmiu/docx#1885] [Q&A] [answered]
I am wondering how to set up my local environment and how to build and import my working copy of the library in said environment. Would like to tackle adding the interactable checkbox and the dropdown list.

**Comment by dolanmiu** [comment, marked as answer]
I don't fully understand the question

checkboxes currently aren't supported yet in docx

**Reply by ric4v1** [reply]
Was asking how can I build the library and import it somewhere else in my local environment, but already figured out how. 
I'm going to try and add the checkboxes, and if successful also dropdown lists. Thanks for the reply !

**Comment by ric4v1** [comment]
Hi @dolanmiu , I forked the project and went ahead and implemented checkboxes. If you want to give it a look and try it out, is the only repo under my user name. Here is a link to it just in case:  https://github.com/ric4v1/docx . Let me know what you think and if you would like to consider adding it to the main project. Regards!

**Reply by dolanmiu** [reply]
excellent, will check it out!

**Reply by dolanmiu** [reply]
Thank you @ric4v1 

Your contribution is in this PR here into `docx`:

https://github.com/dolanmiu/docx/pull/2189

**Reply by ric4v1** [reply]
@dolanmiu perfect! glad to be able to contribute to this amazing library. My next goal is to implement dropdown lists, will update accordingly.

**Reply by dolanmiu** [reply]
Great! 
Looking forward to it!
